client/extensions/woocommerce/app/promotions/fields/style.scss

```diff
@@ -0,0 +1,7 @@
+
+.fields__form-label-required {
+	color: $gray;
```

Comment:
Text (on white) should be at least `$gray-text-min` for good contrast

Reply:
Gotcha, thanks! @kellychoffman or @jameskoster do you have further feedback on the style for the Required tag?